pkg/server/metrics/metrics.go

```diff
@@ -19,6 +19,7 @@ package metrics
 import (
 	"time"
 
+	"k8s.io/klog/v2"
```

Comment:
I think this is an unused import?

Reply:
Yup :D I was initially thrown a little about the way prometheus histogram reporting. I added some logging to understand what it was doing. I removed that logging but not the import it added.